src/ProductlineApp.WebUI/ClientApp/src/components/molecules/table/bodys/AuctionsTableBody.tsx

```diff
@@ -32,47 +29,9 @@ export const AuctionsTableBody = ({
   return (
     <>
       <tbody>
-        <tr className="AuctionsTableRow">
-          <td>
-            <CollapseTableButton isOpen={isOpen} toggle={toggle} />
-          </td>
-          {auctionRecords.map(
-            (auction, key) =>
-              auction &&
-              auction.isActive == showActiveAuctions && (
-                <React.Fragment key={key}>
-                  <td>{auction?.auctionID}</td>
-                  <td>{auction?.sku}</td>
-                  <td>{auction?.brand}</td>
-                  <td>
-                    <div className="productName">
-                      <img className="productImage" src={auction?.productImageUrl}></img>
-                      <p>{auction?.productName}</p>
-                    </div>
-                  </td>
-                  <td>{auction?.category}</td>
-                  <td>{auction?.price} zł</td>
-                  <td>{auction?.quantity}</td>
-                  <td>{auction?.daysToEnd ? auction?.daysToEnd : '-'}</td>
-                  <td>
-                    <div className="auctionsButtonsAction">
-                      <img
-                        className="editAuctionIcon"
-                        src={EditIcon}
-                        onClick={() => onEditAuction(auction.auctionID)}
-                      />
-                      <img
-                        className="backAuctionIcon"
-                        src={BackAuctionIcon}
-                        onClick={() => onWithdrawAuction(auction.auctionID)}
-                      />
-                    </div>
-                  </td>
-                </React.Fragment>
-              ),
-          )}
-        </tr>
-        {isOpen && <CollapseAuctionDetails isOpen={isOpen} auctionRecords={auctionRecords} />}
+        {auctionRecords.map((auction, key) => (
+          <AuctionsTableRow key={key} auction={auction} />
```

Comment:
trzeba przekazac i obsluzyc argumenty "onEditAuction", "onWithdrawAuction", "showActiveAutions" tak jak to bylo wczesniej